### ISSUE TITLE
Accessibility Testing for Veteran Facing forms 21-4138 - Statement in Support of a claim

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#81979]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

[Starting URL for testing](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138)

### Who completed the use of color and color contrast test?

@jeana-adhoc 

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

No issues

### Who completed the axe scans?

@jeana-adhoc 

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

@jeana-adhoc 

### How did axe testing go?

| Axe Results   | Page           |
| ------------- | -------------- |
| Content outside of landmark | [Intro](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction) |
| Content outside of landmark |  [Statement type](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/statement-type) |
| Content outside of landmark |  [Lay witness](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/lay-witness-statement) |
| Content outside of landmark |  [Decision review](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/decision-review) |
| Content outside of landmark | [Select Decision Review](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/select-decision-review) |
| Content outside of landmark | [Supplemental claim](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/supplemental-claim) |
| Content outside of landmark | [HLR](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/higher-level-review) |
| Content outside of landmark |  [Board appeal](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/board-appeal) |
| Content outside of landmark |  [Priority processing](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/about-priority-processing) |
| Content outside of landmark | [Housing risks](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/housing-risks) |
| Content outside of landmark | [Hardships](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/hardships) |
| Content outside of landmark |  [Qualify for priority processing](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/priority-processing-request) |
| Content outside of landmark |  [May not qualify for priority processing](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/priority-processing-not-qualified) |
| Content outside of landmark | [Personal records](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/personal-records-request) |
| Content outside of landmark | [New evidence](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/claim-status-tool) |
| Content outside of landmark |  [Name and Dob](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/personal-information) | 
| Content outside of landmark | [Identification info](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/identification-information) |
| Content outside of landmark |  [Phone & Email](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/contact-information) |
| Content outside of landmark  | [Statement](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/statement) |
|  Content outside of landmark, <br /> remove form title as `<h1>` <br /> Add "Review and Submit" as the `<h1>`,<br /> Headings out of order <br />([ticket #1441](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1441), | [Review and submit](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit) |
|  Content outside of landmark, <br /> remove form title as `<h1>`,<br /> make alert h1,<br /> shift headings accordingly<br />([ticket #1441](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1441), | [Confirmation](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/confirmation) | 

### Who completed the content zoom and reflow test?

@jeana-adhoc 

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

It went well. There's only one minor issue, and I'm not sure if it's worth trying to fix.  

On the new "primary action link",  when the text wraps, the pointy part of the arrow does not scale.

<img width="550" alt="Screenshot 2024-07-08 at 8 13 12 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/bed2aea6-3ae7-4a82-86df-4985ccf3a003">

<img width="550" alt="Screenshot 2024-07-08 at 8 17 03 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/3136a53b-d8af-48f8-ad0c-480c1bba9840">

### Who completed the keyboard navigation test?

@jeana-adhoc 

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

The item that receives focus on each page (which is also the h1) is being announced twice in most cases on mac voiceover.  This is a known issue.  I'll test and see how this behaves in Jaws and NVDA.

#### Minimal header

- The `<header>` tag of the new minimal header is not wrapping the right components. This leaves content hanging out on the page that is not in a defined landmark region that is causing accessibility issues. [I've filed a ticket with the DST.](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2985)

- The VFF team had done its original testing in a mock form 21-0845. It is no longer available on staging, and the sketch files are MIA. However, [looking through old tickets](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/468), I was able to find a screenshot of the annotations for the header part of the form. As the new version is not working as I had originally expected. 

    There are some tweaks that need to be made to match [what was previously approved](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/468):
    1. The stepper needs to be able to have an ID
    2. The h1 needs aria-describedby on it that points to the stepper ID

- In the minimal header, the "back" link acts as a navigational element (like the breadcrumb) and should be in a `<nav>` region, with a `label="Previous page"` on it. Could this end up being an included variant in the va-breadcrumb component?

**Comment by jeana-adhoc** [comment]
Tickets written in response to this audit:

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2985
https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1441
https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1442
https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1443
https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1449
